src/wallet/init.cpp

```diff
@@ -184,8 +184,16 @@ void WalletInit::Construct(InitInterfaces& interfaces) const
         LogPrintf("Wallet disabled!\n");
         return;
     }
-    gArgs.SoftSetArg("-wallet", "");
```

Comment:
Unless I'm missing something, this is breaking change. Why not change the behavior from "load or create each -wallet" to "just load each -wallet"? Then to create a wallet there is the option in the GUI and `createwallet` in RPC.